app/models/theft_alert_plan.rb

```diff
@@ -1,6 +1,12 @@
 class TheftAlertPlan < ActiveRecord::Base
   include Amountable
 
+  LANGUAGE_ENUM = {
```

Comment:
Do you think it makes sense to share this with `Blog`? Even if it's as simple as doing `enum language: Blog::LANGUAGE_ENUM`, I think that would be better than having multiple hashes that are the same thing and have the opportunity to become out of sync. But maybe that doesn't make sense!

Reply:
Good call thanks -- updated in 32879040